README.md

```diff
@@ -1 +1,394 @@
+"math magicians rep" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:  Make simple calculations and Read a random math-related quote.
+
+<!-- <a name="readme-top">The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service (https://www.notion.so/microverse/Leaderboard-API-service-24c0c3c116974ac49488d4eb0267ade3).</a> -->
+
+<h1>math magicians rep</h1>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [math-magicians-rep](#math-magicians-rep)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 math magicians rep](#-math-magicians-rep)
+    - [How to build the "math magicians rep" application?](#how-to-build-the-math-magicians-rep-application)
+    - [Projects list](#projects-list)
+      - [Project 1: Setup.](#project-1-setup)
+      - [Project 2: Components.](#project-2-components)
+      - [Project 3: Events.](#project-3-events)
+      - [Project 4: Fetch data from API.](#project-4-fetch-data-from-api)
+      - [Project 5: Full website.](#project-5-full-website)
+      - [Project 6: Tests (pair programming).](#project-6-tests-pair-programming)
+      - [Deployment.](#deployment)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Tools i have used for this project ](#tools-i-have-used-for-this-project-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment-1)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+# 📖 math magicians rep<a name="about-project"></a>
+It is a Single Page App (SPA) that allows users to:  Make simple calculations and Read a random math-related quote.
+
+<!-- > Describe your project in 1 or 2 sentences. -->
+### How to build the "math magicians rep" application?
+I will start by building the core functionality, which is a calculator. At the very beginning my website will contain only the simple UI presented below, but it will be able to perform all math operations.
+<p align="center">
+  <img src="./public/assets/calculator.png" alt="Calculator" width="300px" />
+</p>
+
+Once the calculator is ready I will use it as a part of a full website that includes 3 sub-pages:
+<p align="center">
+  <img src="./public/assets/math_magicians_home.png" alt="Core elements" width="450px"  />
+  <img src="./public/assets/math_magicians_calculator.png" alt="Core elements" width="450px"  />
+  <img src="./public/assets/math_magicians_quote.png" alt="Core elements" width="450px"  />
```

Comment:
The link to your images is currently broken, kindly fix them as this gives your readme a less professional look😉.
<img width="647" alt="image" src="https://github.com/fickryiman/math-magicians-rep/assets/64643995/9bc78f46-c70d-4c20-b752-028c51b12425">